website/src/components/Chat/ChatSection.tsx

```diff
@@ -21,14 +21,18 @@ export const ChatSection = ({ chatId }: { chatId: string }) => {
     },
   });
 
-  useCacheChatForm(form);
+  // useCacheChatForm(form);
```

Comment:
I disable this for now because it cause a hydration error.

Reply:
I still had the error, so I also removed the other part